### PR TITLE
RLM-1088 Add docs for running tests on an MNAIO

### PR DESCRIPTION
**Pull request by antonym**
Adds docs for testing an MNAIO leapfrog using gating scripts.

### CODE REVIEW

File: testing.rst [hunk 1]
```diff
@@ -26,10 +33,36 @@ you wish to test with.
...
+Build an OnMetal server using the flavor onmetal-io1 and running
+Ubuntu 16.04 LTS.
+
+You'll need to set the RE_JOB_IMAGE to xenial_mnaio in order to
+trigger the mnaio code path.  Then set the action to set the
+leap you want to test.
+
+.. code-block:: shell
+
+    export RE_JOB_IMAGE=xenial_mnaio
```
Comment (claco) [inline testing.rst]: Does this tell o-a-ops to spin it's vms as xenial vms?
If so, shouldn't this be trusy_mnaio?
If not, maybe some clarifying explanation.

Comment (antonym) [reply testing.rst]: Yeah, my bad... I've been in xenial mode this week, I'll correct.

Comment (antonym) [reply testing.rst]: Eh, actually that's correct... we're using a Xenial Host now for MNAIO testing, the image that it pulls down will be Trusty though.

Comment (antonym) [reply testing.rst]: This is where you can tune the gating script to use Trusty or Xenial as the VM.  Right now the assumption is all leapfrogs start from Trusty: 

https://github.com/rcbops/rpc-upgrades/blob/master/tests/create-mnaio.sh#L39

Comment (claco) [reply testing.rst]: Cool. Thanks for the clarification. We might want to just dump that comment/blurb/knowledge into this doc so the next person reading it knows that too.

Comment (antonym) [reply testing.rst]: Added a quick blurb about it.

**Pull request merged**